programs/mango-v4/src/instructions/token_register.rs

```diff
@@ -91,7 +91,9 @@ pub fn token_register(
         force_close: 0,
         padding: Default::default(),
         fees_withdrawn: 0,
-        reserved: [0; 2104],
+        token_conditional_swap_taker_fee_fraction: 0.0,
```

Comment:
just to make sure I get this - so you went with 5 bps for trustless and 0 for trusted

Reply:
My current idea is that "register" leaves most things null and expects a follow up with "edit", whereas "register_trustless" should come with full defaults.

Probably better if register also comes with defaults.